db/migrate/20150726062900_create_course_assessment_question_multiple_response.rb

```diff
@@ -0,0 +1,49 @@
+class CreateCourseAssessmentQuestionMultipleResponse < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [16/15]<br>Method has too many lines. [41/10]